ui/src/app/applications/components/utils.tsx

```diff
@@ -296,31 +296,31 @@ export function findChildPod(node: appModels.ResourceNode, tree: appModels.Appli
     });
 }
 
-export const deletePodAction = async (pod: appModels.Pod, appContext: AppContext, appName: string, appNamespace: string) => {
-    appContext.apis.popup.prompt(
+const deletePodAction = async (ctx: ContextApis, pod: appModels.ResourceNode, app: appModels.Application) => {
+    ctx.popup.prompt(
         'Delete pod',
```

Comment:
Should this delete pod popup have the same icon and yellow color as the other delete resource popups, just for consistency?

Reply:
No. The goal of this popup is to ask for confirmation like a "normal" action. The reasoning behind this is that managed pods are likely to be recreated automatically. Deleting a pod should not cause an outage of an application as Pod deletion is expected in Kubernetes. 

If you compare it to a ReplicaSet or Deployment for instance, the impact of deleting such resources could create outages, even if they end up being recreated, hence the difference in the pupup reflecting the risk level.

Reply:
I see your point.  The reason is more of consistency in presentation in general for any Delete action and it's less about the resulting outcome and severity of the action.  For example, the user could have inadvertently or accidentally selected this menu item action.  This accidental selection is one of the reasons why a warning popup is needed.

For those other more detrimental outcomes, forcing the user to type in the resource/app name was added.  

If we find that this is still not sufficient, then perhaps the popup should be more of a Caution or Stop indicator? (It's no longer a Warning).

Reply:
From the meeting today the consensus is this isn't a show stopper and we should go ahead and merge but it would be great to get a different icon so that it's more accessible.